# Type: fix
# Subtype: I2
# English Summary: Change the path of size.html.erb
# Chinese Summary: 更改size.html.erb的路径
# Entry Number: 23

# === Git Diff Content ===

diff --git a/plugins/sonar-core-plugin/src/main/java/org/sonar/plugins/core/widgets/SizeWidget.java b/plugins/sonar-core-plugin/src/main/java/org/sonar/plugins/core/widgets/SizeWidget.java
index 0df4940867..b3330c7833 100644
--- a/plugins/sonar-core-plugin/src/main/java/org/sonar/plugins/core/widgets/SizeWidget.java
+++ b/plugins/sonar-core-plugin/src/main/java/org/sonar/plugins/core/widgets/SizeWidget.java
@@ -22,6 +22,6 @@ package org.sonar.plugins.core.widgets;
 public class SizeWidget extends CoreWidget {
 
   public SizeWidget() {
-    super("size", "Size metrics", "/Users/julienlancelot/Dev/Sources/sonar/plugins/sonar-core-plugin/src/main/resources/org/sonar/plugins/core/widgets/size.html.erb");
+    super("size", "Size metrics", "/org/sonar/plugins/core/widgets/size.html.erb");
   }
 }
